app/src/main/java/com/firebase/ecommerce/feature_login/presentation/screens/LoginScreen.kt

```diff
@@ -357,10 +481,9 @@ fun ButtonSign(
 }
 
 class NoRippleInteractionSource : MutableInteractionSource {
```

Comment:
@AishwaryaAishu12 Are you using this class anywhere, if yes please maintain this in seperate file